frappe/public/js/frappe/form/formatters.js

```diff
@@ -50,7 +50,7 @@ frappe.form.formatters = {
 		return frappe.form.formatters._right(value==null ? "" : cint(value), options)
 	},
 	Percent: function(value, docfield, options) {
-		return frappe.form.formatters._right(flt(value, 2) + "%", options)
+		return frappe.form.formatters._right(flt(value, frappe.defaults.get_default("float_precision") || 3) + "%", options);
```

Comment:
```suggestion
              var precision = docfield.precision
			|| cint(frappe.boot.sysdefaults && frappe.boot.sysdefaults.float_precision)
			|| 2;
		return frappe.form.formatters._right(flt(value, precision) + "%", options);
```
we fall back to 2 , because it was that way before, lets not break things perhaps. consider docfied precision too. something like above.